client/reader/following-manage/index.jsx

```diff
@@ -140,6 +142,25 @@ class FollowingManage extends Component {
 		);
 	};
 
+	reportFollowByUrlRender = () => {
+		const siteUrl = this.props.readerAliasedFollowFeedUrl;
+		const showingFollowByUrlButton = this.shouldShowFollowByUrl();
+
+		if ( siteUrl && showingFollowByUrlButton ) {
+			recordTrack( 'calypso_reader_following_manage_follow_by_url_render', {
+				url: siteUrl,
+			} );
+		}
+	};
+
+	shouldShowFollowByUrl = () => resemblesUrl( this.props.sitesQuery );
```

Comment:
Didn't we stop using `resemblesUrl` somewhere along the line, and just trigger follow by URL based on a http:// or https:// prefix?